client/blocks/reader-full-post/index.jsx

```diff
@@ -333,6 +335,38 @@ export class FullPostView extends Component {
 		}
 	};
 
+	trackFastExit = ( post, elapsedTime, fastExitThreshold ) => {
+		recordTrackForPost( 'calypso_reader_article_fast_exit', post, {
+			context: 'full-post',
+			estimated_reading_time: post.minutes_to_read,
+			elapsed_time: elapsedTime,
```

Comment:
Will there be any issues because estimated_reading_time is in minutes and elasped_time is in seconds?

Reply:
I'll rename the `elasped_time` to `elasped_seconds` to make it clearer